website/docs/_advanced_install.mdx

```diff
@@ -330,6 +334,46 @@ sdk install scalacli
 </Tabs>
 
 </TabItem>
+
+<TabItem value="freebsd">
+<Tabs
+groupId="freebsd"
+defaultValue="manual"
+values={[
+{label: 'Manual', value: 'manual'},
+]}
+>
+
+<TabItem value="manual">
+FreeBSD support (and support for other BSDs) is currently very partial. Scala CLI does not distribute for FreeBSD, but you can install it via Coursier's standalone JAR.
+
+Before running Coursier and installing Scala CLI, make sure you have Java 17 or higher installed and in your $PATH:
+```bash
+pkg install openjdk17
+```
+Download Coursier via the [Any JAR option](https://get-coursier.io/docs/cli-installation#latest-launchers).
+
+After downloading Coursier, you can use it to install Scala CLI:
+```console
+chmod +x coursier
+./coursier install scala-cli
+```
+You now have scala-cli installed in your system. Scala CLI is installed as a JAR rather than a native binary, and so it has a slightly higher startup time.
+
+To run scala-cli anywhere on your system, you need to add its binary path to the $PATH variable. Assuming your user directory is **foobar**:
+```bash
+export PATH="$PATH:/home/foobar/.local/share/coursier/bin"
+```
+Add this command to your shell's RC file to make the setting permanent.
+
+To build your projects successfully, remember to add ```--server=false``` to your CLI arguments, otherwise Scala CLI fails to work on FreeBSD.
```

Comment:
````suggestion
Depending on the Java distribution you are using, it has been observed that Bloop (the build server used by Scala CLI) may have trouble starting on FreeBSD. You may try to customise Bloop JVM options with the `BLOOP_JAVA_OPTS` environment variable. Alternatively, you may skip using the build server altogether and use the compiler directly by passing the `--server=false` option. Please note that doing this will noticably extend project build times.

```bash
scala-cli run -e 'println("Hello FreeBSD")' --server=false
```
````